### ISSUE TITLE
missing shows, possible app change

### ISSUE THREAD

**Issue by bobbintb** [turicfr/nick-downloader#2]
I think they changed something in the app because the script now shows far fewer shows than the app does.

**Comment by turicfr** [comment]
Hey, thank you for reporting an issue.

Can you please provide more information? like from which server are you trying to download (America or Europe and which country), which shows are missing from the script?

**Comment by bobbintb** [comment]
I am using the US version. I'm not sure which shows but there are a lot of them. Your script shows 9 shows but the app shows 29. I know Alvin and the Chipmunks is one of them. I thought maybe it is just filtering shows that have no episodes but I checked and Alvin does have episodes. That's the only one I checked because there are too many. I checked the app traffic with Wireshark and Bluestacks and I didn't see any of the api information the script uses, no MGID or the api.playplay.viacom address. I might have missed it though.
